src/Components/ExternalResult/ListFilter.tsx

```diff
@@ -64,6 +60,13 @@ export default function ListFilter(props: any) {
     setSelectedLsgs(value);
   };
 
+  const field = (name: string) => ({
+    name,
+    label: t(name),
+    value: filterState[name],
+    onChange: handleChange,
```

Comment:
None of the filters have error messages. Also the sapcing between the fields are too high, statically hiding the error span would do the trick.

```suggestion
    onChange: handleChange,
    errorClassName="hidden",
```

Reply:
Nice catch!